vocabularies/geocoding.ttl

```diff
@@ -14,6 +14,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <http://opengis.net/ont/geocode/GeoHash>
     a
         <http://opengis.net/ont/geocode/GeoHashCode> ,
+        skos:inScheme <http://opengis.net/ont/geocode/GeoCodes> ;

```

Comment:
@situx you inserted "inScheme" between the two RDF types, so now the syntax is invalid.